marquee/marquee.ino

```diff
@@ -87,7 +84,14 @@ static const char WEB_ACTION3[] PROGMEM = "</a><a class='w3-bar-item w3-button'
                        "<a class='w3-bar-item w3-button' href='https://github.com/Qrome/marquee-scroller' target='_blank'><i class='fas fa-question-circle'></i> About</a>";
 
 static const char CHANGE_FORM1[] PROGMEM = "<form class='w3-container' action='/locations' method='get'><h2>Configure:</h2>"
-                      "<label>TimeZone DB API Key (get from <a href='https://timezonedb.com/register' target='_BLANK'>here</a>)</label>"
+                      "<label>WagFam Calendar Data Source</label>"
+                      "<input class='w3-input w3-border w3-margin-bottom' type='text' name='wagFamDataSource' value='%WAGFAMDATASOURCE%' maxlength='256'>"
+                      "<label>WagFam Calendar API Key</label>"
+                      "<input class='w3-input w3-border w3-margin-bottom' type='text' name='wagFamApiKey' value='%WAGFAMAPIKEY%' maxlength='128'>"
+                      "<p><input name='enwagfam' class='w3-check w3-margin-top' type='checkbox' %WAGFAM_ENABLED%> Enable WagFam Calendar</p>"
+                      "<hr>";
+
+static const char CHANGE_FORM2[] PROGMEM = "<label>TimeZone DB API Key (get from <a href='https://timezonedb.com/register' target='_BLANK'>here</a>)</label>"
```

Comment:
Here I wanted our new config at the top, so I added a new variable for our section of the form, put it at the top, and shifted all of the previous config items below it.